docs/quickstart/installation.md

```diff
@@ -63,7 +58,7 @@ pip install torch==2.0.1+cu118 torchvision==0.15.2+cu118 --extra-index-url https
 
 - To install 2.0.1 with CUDA 11.7:
 
-Note that if a pytorch version prior to 2.0.1 is installed, 
+Note that if a pytorch version prior to 2.0.1 is installed,
```

Comment:
My understanding is that functorch may need to be explicitly uninstalled
`pip uninstall functorch`
because just uninstalling torch doesn't seem to uninstall functorch.
However, that may also impact other projects.